{
    "CVE-ID": "CVE-2022-48560",
    "NVD": {
        "Descriptions": "A use-after-free exists in Python through 3.9 via heappushpop in heapq.",
        "Cvss2_BaseScore": 0,
        "Cvss3_BaseScore": 7.5,
        "Cvss2_VectorString": "",
        "Cvss3_VectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
        "CPEs": [
            {
                "URI": "cpe:/a:python:python",
                "FormattedString": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "3.7.0",
                "VersionEndExcluding": "",
                "VersionEndIncluding": "3.9.0"
            }
        ]
    },
    "Official": {
        "3.6.11rc1": {
            "source": [
                "https://github.com/python/cpython/blob/3.6/Misc/NEWS.d/3.6.11rc1.rst"
            ],
            "BTS": {
                "bpo39421": {
                    "ID": "bpo39421",
                    "url": "https://bugs.python.org/issue39421",
                    "date": "2020-01-22-15-53-37",
                    "nonce": "O3nG7u",
                    "section": "Core and Builtins"
                }
            }
        },
        "3.7.7rc1": {
            "source": [
                "https://github.com/python/cpython/blob/3.7/Misc/NEWS.d/3.7.7rc1.rst"
            ],
            "BTS": {
                "bpo39421": {
                    "ID": "bpo39421",
                    "url": "https://bugs.python.org/issue39421",
                    "date": "2020-01-22-15-53-37",
                    "nonce": "O3nG7u",
                    "section": "Core and Builtins"
                }
            }
        },
        "3.8.2rc1": {
            "source": [
                "https://github.com/python/cpython/blob/v3.8.2rc1/Misc/NEWS.d/3.8.2rc1.rst"
            ],
            "BTS": {
                "bpo39421": {
                    "ID": "bpo39421",
                    "url": "https://bugs.python.org/issue39421",
                    "date": "2020-01-22-15-53-37",
                    "nonce": "O3nG7u",
                    "section": "Core and Builtins"
                }
            }
        },
        "3.9.0a3": {
            "source": [
                "https://github.com/python/cpython/blob/3.12/Misc/NEWS.d/3.9.0a3.rst"
            ],
            "BTS": {
                "bpo39421": {
                    "ID": "bpo39421",
                    "url": "https://bugs.python.org/issue39421",
                    "date": "2020-01-22-15-53-37",
                    "nonce": "O3nG7u",
                    "section": "Core and Builtins"
                }
            }
        }
    }
}